### DISCUSSION TITLE
radio显示bug，未解决，谢谢

### DISCUSSION THREAD

**Discussion by caixianglin** [alibaba/formily#2594] [Q&A 2.x]
demo: https://codesandbox.io/s/stoic-hertz-8gob2?file=/App.tsx

单独radio组件正常
自增组件嵌套radio，一选择单选，整个视图消失
![image](https://user-images.githubusercontent.com/10279815/145174523-77d3515a-03bc-4e0a-94d7-cb1b4c288ea2.png)
选择2，空白
![image](https://user-images.githubusercontent.com/10279815/145174574-652a7501-12f9-4653-968d-1815f8c50036.png)

**Comment by janryWang** [comment]
```
type: "object",
 "x-component": "ArrayCollapse.CollapsePanel" 内部再套一层void，指定x-component为ArrayCollapse.CollapsePane

```

**Reply by caixianglin** [reply]
无效啊，是这样吗
![image](https://user-images.githubusercontent.com/10279815/145320513-9f2a2f48-c270-4fa4-8890-03ecdbc5a787.png)

**Reply by caixianglin** [reply]
上面有codesandbox的demo，可以实时修改，多谢

**Reply by janryWang** [reply]
的确有这个bug，麻烦提一个issue，我这边修一下

**Reply by caixianglin** [reply]
ok
